### PR TITLE
Add black for Python

### PR DESCRIPTION
**Pull request by sgaist**
This merge requests adds black for Python to the tools used when running the pep8 compliance tests.

It follows the morepath/reg project so that they now share their test infrastructure more.

The code has been then passed through black.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[flake8]
+show-source = True
+ignore = E203, E731, W503
+max-line-length = 88
```
Comment (jugmac00) [inline setup.cfg:4]: Here is a line length of 88, but in pyproject.toml a line length of 80 is specified.

Is this intended? Maybe line length for black is not a hard limit?

Or was this accidental?

**Review by jugmac00 — APPROVED** [review]
LGTM - with one annoation

File: setup.cfg (hunk 1 above)
Comment (jugmac00) [reply setup.cfg:4]: As explained to me in the other project, this seems to be best practice and even necessary as the value is no hard limit for black.

**Comment by sgaist** [comment]
Small ping :-)

**Pull request merged**